components/net/http_loader.rs

```diff
@@ -1290,8 +1290,32 @@ fn http_network_fetch(
         }
     }
 
+    let header_strings: Vec<&str> = res
+        .headers()
+        .get_all("Timing-Allow-Origin")
+        .iter()
+        .map(|header_value| header_value.to_str().unwrap())
```

Comment:
Let's use unwrap_or("") instead, to avoid any issues with unusable header values causing a panic.